src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Constructors.jl

```diff
@@ -139,11 +139,20 @@ julia> affine_space(QQ,[:y1,:z2,:a])
 Spec of Multivariate Polynomial Ring in y1, z2, a over Rational Field
 ```
 """
-function affine_space(kk::BRT, var_symbols::Vector{Symbol}) where {BRT<:Ring}
+function affine_space(kk::BRT, var_symbols::Vector{Symbol}) where {BRT<:Field}
```

Comment:
ok, you disallow rings like ZZ here. But where is then the difference between lines 142--145 and 152--155?